Add 'tartiflette', 'tartiflette-aiohttp' and 'tartiflette-asgi' for Python 3.6. (#1252)
vinta/awesome-python: README.md

* [urwid](http://urwid.org/) - A library for creating terminal GUI applications with strong support for widgets, events, rich colors, etc.
* [wxPython](https://wxpython.org/) - A blending of the wxWidgets C++ class library with the Python.

## GraphQL

*Libraries for working with GraphQL.*

> * [tartiflette](https://tartiflette.io) - GraphQL Implementation, SDL First, for python 3.6+ / asyncio.

Reviewer: ```suggestion
* [tartiflette](https://tartiflette.io) - SDL-first GraphQL engine implementation for Python 3.6+ and asyncio.
```

> * [tartiflette-aiohttp](https://github.com/tartiflette/tartiflette-aiohttp/) - Wrapper of tartiflette to expose GraphQL API over HTTP based on aiohttp / 3.6+ / asyncio.

Reviewer: ```suggestion
* [tartiflette-aiohttp](https://github.com/tartiflette/tartiflette-aiohttp/) - An `aiohttp`-based wrapper for Tartiflette to expose GraphQL APIs over HTTP.
```

> * [tartiflette-asgi](https://github.com/tartiflette/tartiflette-asgi/) - a wrapper that provides ASGI support for the Tartiflette Python GraphQL engine.

Reviewer: ```suggestion
* [tartiflette-asgi](https://github.com/tartiflette/tartiflette-asgi/) - ASGI support for the Tartiflette GraphQL engine.
```
